api/base/middleware.py

```diff
@@ -156,3 +159,31 @@ def process_response(self, request, response):
                 httponly=settings.CSRF_COOKIE_HTTPONLY,
             )
         return response
+
+
+class WaffleMiddleware(MiddlewareMixin):
+    """
+    This must be overriden for our domain rules and to ensure the cookies never expire,
+    """
+    def process_response(self, request, response):
+        secure = get_setting('SECURE')
+
+        if request.environ['SERVER_NAME'] == 'localhost':
```

Comment:
@mfraezz Do you know if this handles custom domains?

Reply:
`'localhost'`? I would assume this is for local development and tests. Unsure about `request.environ`

Reply:
I mean is SERVER_NAME going to reflect the domain name, not including sub domains, on staging, custom domains etc.